subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/JvmProjectInitDescriptor.java

```diff
@@ -174,9 +175,15 @@ private void addJcenter(BuildScriptBuilder buildScriptBuilder) {
         buildScriptBuilder.repositories().jcenter("Use JCenter for resolving dependencies.");
     }
 
-    private void addStandardDependencies(BuildScriptBuilder buildScriptBuilder) {
+    private void addStandardDependencies(BuildScriptBuilder buildScriptBuilder, boolean constraintsDefined) {
+        String commonsTextVersion = libraryVersionProvider.getVersion("commons-text");
+        String commonsTextCoordinates = constraintsDefined ? "org.apache.commons:commons-text" : "org.apache.commons:commons-text:" + commonsTextVersion;
+        buildScriptBuilder.implementationDependency(null, commonsTextCoordinates);
+
```

Comment:
```suggestion
```

:x: This whole thing should go, as it adds the "commons-text" dependency everywhere, which is not what we want.

I think we should add some test. For example a test that asserts the runtime classpath of the application of a generated application (using the `dependencies` task for example). So that we don't miss issues if extend the example in the future.